### ISSUE TITLE
Config volume

### ISSUE THREAD

**Issue by nemchik** [manbearwiz/youtube-dl-server#24]
I've been using a volume to make `/etc/youtube-dl.conf` persistent, but it would be great if that file could be read from a `/config` volume. Maybe other config related files would end up in there as well if there are any? It seems like a bad idea to use `/etc` itself as a volume.

**Comment by GlassedSilver** [comment]
Hear hear. youtube-dl is all about configuring it to your needs. Pretty drastic oversight and looking at the date of this issue I'm quite baffled... This has been unaddressed for 5 months now?

**Comment by crosbyhayton** [comment]
I agree, it would be great to be able to set options using a standard config file.
